src/libraries/System.Runtime.InteropServices.JavaScript/gen/JSImportGenerator/Constants.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.Interop.JavaScript
+{
+    internal static class Constants
+    {
+        public const int JavaScriptMarshalerArgSize = 16;
```

Comment:
This seems like an deviously simple constant, but I think this is the ABI for the marshaller. What are its units? Also, does it need to be 16 bytes? or N pointer sizes? If changing this can break things a comment is likely warranted.

Reply:
That's obsolete code now, will delete it. Thanks!